guides/common/modules/ref_system-requirements.adoc

```diff
@@ -44,6 +44,12 @@ endif::[]
 If your territory is USA and your language is English, set `en_US.utf-8` as the system-wide locale settings.
 For more information about configuring system locale in {EL}, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/assembly_changing-basic-environment-settings_configuring-basic-system-settings#proc_configuring-the-system-locale_assembly_changing-basic-environment-settings[Configuring System Locale guide].
 
+ifeval::["{context}" == "{project-context}"]
```

Comment:
Is this Satellite-only?

Reply:
Considering the Subscription management, I guess it should be Satellite-specific.

Reply:
Yes, you're absolutely correct. It's not about a RHEL subscription, but the satellite subscription.

My question was mostly because I didn't fully understand the "ifeval" statement. I meant to hint towards maybe changing this to "ifdef::satellite"; or at least add it as additionl layer if necessary. right now, as you can see with the list of changed files from the "github-actions" bot, it also affects other build targets.

Reply:
Yes. Switched to `ifdef` :+1: